src/logic/contracts/safeContracts.ts

```diff
@@ -11,7 +11,7 @@ import { isProxyCode } from 'src/logic/contracts/historicProxyCode'
 import Web3 from 'web3'
 
 export const SENTINEL_ADDRESS = '0x0000000000000000000000000000000000000001'
-export const MULTI_SEND_ADDRESS = '0xB522a9f781924eD250A11C54105E51840B138AdD'
+export const MULTI_SEND_ADDRESS = '0x8d29be29923b68abfdd21e541b9374737b49cdad'                                   
```

Comment:
Why all the whitespaces?

Reply:
Also can we please use the checksummed version? https://github.com/gnosis/safe-contracts/blob/development/networks.json#L1446

Reply:
sometimes vscode do me that. Fixed!